index.js

```diff
@@ -95,8 +99,8 @@ const parseArgs = (
   }
 
   const result = {
-    flags: {},
-    values: {},
+    flags: new Map(),
+    values: new Map(),
```

Comment:
These need to use SafeMap instead - and, then they don’t have to use the prototype primordials.

Reply:
_(@shadowspawn note, you can just import SafeMap from primordials)._